extension/src/setup/studio.ts

```diff
@@ -0,0 +1,129 @@
+import { EventEmitter } from 'vscode'
+import { isStudioAccessToken } from './token'
```

Comment:
I wanted to move `isStudioAccessToken` into `Studio` but it's being used in `setup/InputBox`.

Reply:
it could still be a function in this file

Reply:
Oh, right, I've seen that pattern in other modules as well!